src/py/flwr/dataset/utils/common.py

```diff
@@ -205,3 +205,100 @@ def adjust_y_shape(nda: np.ndarray) -> np.ndarray:
     """Turn shape (x, 1) into (x)."""
     nda_adjusted = np.reshape(nda, (nda.shape[0]))
     return cast(np.ndarray, nda_adjusted)
+
+
+def create_dla_partitions(
```

Comment:
Shouldn't this function rather be named `create_lda_partitions`?

Reply:
Yep, changed to reflect the correct name.